niceml/dlframeworks/tensorflow/metrics/objdetmetrics.py

```diff
@@ -13,10 +13,11 @@ class AvgPosPredObjDet:  # pylint: disable=too-few-public-methods
     """Positive Classification Values for object detection"""
 
     def __init__(self, name: str = "avg_pos_pred"):
-
+        """Constructor"""
```

Comment:
Please add more details. I think it's fine if you use the class docstring in combination with a description of the attributes.